seguro/common/job.py

```diff
@@ -0,0 +1,19 @@
+# SPDX-FileCopyrightText: 2023 Felix Wege, EONERC-ACS, RWTH Aachen University
```

Comment:
This should be you, not me :wink:

Reply:
Fixed in https://github.com/SEGuRo-Projekt/Platform/pull/76/commits/0c9a48b4e98887b2e86e82b6818bb7823a84a282